mypy issue on master

```bash
Run bash ./tests/run_code_style.sh mypy
+ '[' mypy = lint ']'
+ '[' mypy = fmt ']'
+ '[' mypy = mypy ']'
+ mypy --config-file mypy.ini
ignite/distributed/comp_models/native.py:125: error: Argument 3 to
"init_process_group" has incompatible type "**Dict[str, int]"; expected
"timedelta"  [arg-type]
    ...init_process_group(backend, init_method=init_method, **init_pg_kwargs)
                                                              ^
ignite/distributed/comp_models/native.py:125: error: Argument 3 to
"init_process_group" has incompatible type "**Dict[str, int]"; expected
"Optional[Store]"  [arg-type]
    ...init_process_group(backend, init_method=init_method, **init_pg_kwargs)
                                                              ^
ignite/distributed/comp_models/native.py:125: error: Argument 3 to
"init_process_group" has incompatible type "**Dict[str, int]"; expected "str" 
[arg-type]
    ...init_process_group(backend, init_method=init_method, **init_pg_kwargs)
                                                              ^
Found 3 errors in 1 file (checked 103 source files)
```

--- a/ignite/distributed/comp_models/native.py
+++ b/ignite/distributed/comp_models/native.py
@@ -110,7 +110,7 @@ if has_native_dist_support:
             self._backend = backend
             self.setup_env_vars(rank, world_size)
 
-            init_pg_kwargs = {}
+            init_pg_kwargs: Dict[str, Any] = {}
             if timeout is not None:
                 init_pg_kwargs["timeout"] = timeout
 

